core/App/ehsm_napi.cpp

```diff
@@ -1549,11 +1550,11 @@ char* NAPI_GenerateQuote(const char *challenge)
  *          }
  *      }
  */
-char* NAPI_VerifyQuote(const char *quote_base64, const char *nonce)
+char* NAPI_VerifyQuote(const char *quote_base64, const char *mr_signer_base64, const char *mr_enclave_base64, const char *nonce_base64)
```

Comment:
i'm not mean mr_signer_base64, i want to change it  as mrsigner_base,  which means the base value of the mrsigner

Reply:
Ok, I will change it.

Reply:
may be change the name to mrenclave_base_base64? because the char* is base64 string